licenses.yaml

```diff
@@ -2279,6 +2308,16 @@ libraries:
 
 ---
 
+name: Apache Log4j
```

Comment:
Do we need log4j? (Normally we use log4j2 + slf4j bindings to get other stuff adapted into log4j2)

I would expect to see `slf4j-log4j12`, I think.

Reply:
`mvn dependency:tree` shows that it's being pulled in by the `com.101tec:zkclient dependency`:
```
[INFO] +- com.101tec:zkclient:jar:0.3:compile
[INFO] |  \- log4j:log4j:jar:1.2.14:compile
```
I'll exclude it.